parser/digester_test.go

```diff
@@ -34,6 +34,9 @@ func TestNormalize(t *testing.T) {
 		{"select \\N", "select ?"},
 		{"SELECT `null`", "select `null`"},
 		{"select * from b where id = 1", "select * from `b` where `id` = ?"},
+		{"select * from t where a in (1)", "select * from `t` where `a` in ( ... )"},
```

Comment:
Please add 2 more test cases:
`select * from t where (a, b) in ((1, 1))`
`select * from t where (a, b) in ((1, 1), (2, 2))`